shared/src/clients/mail-client.class/index.ts

```diff
@@ -76,6 +76,10 @@ export default class MailClient {
         headers,
```

Comment:
Can just add `attachments: input.attachments,` here instead of having an if condition as they means the same thing

Reply:
sure, I thought the `if` condition made it clearer that this was optional but I don't feel strongly

Reply:
[6b4ba8c](https://github.com/opengovsg/postmangovsg/pull/1731/commits/6b4ba8cf80e66c71cb5af39d4085da723bffc50b)